core/src/test/scala/org/apache/spark/deploy/history/HistoryServerSuite.scala

```diff
@@ -632,6 +632,49 @@ class HistoryServerSuite extends SparkFunSuite with BeforeAndAfter with Matchers
     }
   }
 
+  test("SPARK-28564: Access history application defaults to the last attempt id") {
```

Comment:
Hi, @cxzl25 .
Since you created a JIRA issue with `Improvement` type, we don't use JIRA ID in the test case name.
That's because we don't add JIRA ID prefix for all new features.

Reply:
ok, I remember now. thanks for your reminder